README.md

```diff
@@ -13,3 +13,159 @@
 
 GraphQL Service runs on port 23431 at `/graphql` by default
 
+
+## Queries
+Here are some of the important GraphQL queries:
+
+### 1. Get all traces in provided time range
+
+```graphql
+curl -s localhost:2020/graphql -H 'Content-Type: application/graphql' -d\
+'{
+  traces(
+    type: API_TRACE
+    limit: 10
+    between: {
+      startTime: "2015-01-01T00:00:00.000Z"
+      endTime: "2025-01-01T00:00:00.000Z"
+    }
+  ) {
+    results {
+      id
+    }
+  }
+}'
+```
+
+### 2. Find trace using TraceID
+
+```graphql
+curl -s localhost:2020/graphql -H 'Content-Type: application/graphql' -d \
+'{
+  traces(
+    type: API_TRACE
+    between: {
+      startTime: "2015-01-01T00:00:00.000Z"
+      endTime: "2025-01-01T00:00:00.000Z"
+    }
+    filterBy: [
+      {
+        operator: EQUALS
+        value: "348bae39282251a5"
+        type: ID
+        idType: API_TRACE
+      }
+    ]
+  ) {
+    results {
+      id
```

Comment:
If there's no way to return the canonical fields used to represent spans, we may want to remove this or rename it to "verify trace exists"